VideoLocker/src/main/java/org/edx/mobile/util/UnicodeChars.java

```diff
@@ -0,0 +1,11 @@
+package org.edx.mobile.util;
+
+
+public class UnicodeChars {
+    // Make this class non-instantiable
+    private UnicodeChars() {
+        throw new UnsupportedOperationException();
+    }
+
+    public static final char NON_BREAKING_HYPHEN = '\u2011';
```

Comment:
Maybe add a javadoc here as well. Although the name is self-evident here, in other cases we might need more explanation.